pipeline/signed-compose-listener.groovy

```diff
@@ -9,109 +9,130 @@ def platform
-                "nvr": "RHCEPH-${versions.major_version}.${versions.minor_version}",
-                "phase": "rc",
-                "type": "product-build",
-                "version": cephVersion
-            ],
-            "build": [
-                "platform": platform,
-                "compose-url": composeUrl
-            ],
-            "contact": [
-                "email": "ceph-qe@redhat.com",
-                "name": "Downstream Ceph QE"
-            ],
-            "run": [
-                "log": "${env.BUILD_URL}console",
-                "url": env.BUILD_URL
-            ],
-            "version": "1.0.0"
-        ]
-        def msgContent = writeJSON returnText: true, json: contentMap
-        def overrideTopic = "VirtualTopic.qe.ci.rhcephqe.product-build.update.complete"
+        }
 
-        sharedLib.SendUMBMessage(msgContent, overrideTopic, "ProductBuildDone")
-        println "Updated UMB Message Successfully"
+        stage('Messaging') {
+            def contentMap = [
+                "artifact": [
+                    "build_action": "rc",
+                    "name": "Red Hat Ceph Storage",
+                    "nvr": "RHCEPH-${versions.major_version}.${versions.minor_version}",
+                    "phase": "rc",
+                    "type": "product-build",
+                    "version": cephVersion
+                ],
+                "build": [
+                    "platform": platform,
+                    "compose-url": composeUrl
+                ],
+                "contact": [
+                    "email": "ceph-qe@redhat.com",
+                    "name": "Downstream Ceph QE"
+                ],
+                "run": [
+                    "log": "${env.BUILD_URL}console",
+                    "url": env.BUILD_URL
+                ],
+                "version": "1.0.0"
+            ]
+            def msgContent = writeJSON returnText: true, json: contentMap
+            def overrideTopic = "VirtualTopic.qe.ci.rhcephqe.product-build.update.complete"
+
+            sharedLib.SendUMBMessage(msgContent, overrideTopic, "ProductBuildDone")
+            println "Updated UMB Message Successfully"
+        }
+    } catch(Exception err) {
+        if (currentBuild.result != "ABORTED") {
+            // notify about failure
+            currentBuild.result = "FAILURE"
+            def failureReason = err.getMessage()
+            def subject =  "[CEPHCI-PIPELINE-ALERT] [JOB-FAILURE] - ${env.JOB_NAME}/${env.BUILD_NUMBER}"
+            def body = "<body><h3><u>Job Failure</u></h3></p>"
+            body += "<dl><dt>Jenkins Build:</dt><dd>${env.BUILD_URL}</dd>"
+            body += "<dt>Failure Reason:</dt><dd>${failureReason}</dd></dl></body>"
+
+            emailext (
+                mimeType: 'text/html',
+                subject: "${subject}",
+                body: "${body}",
+                from: "cephci@redhat.com",
+                to: "ceph-qe-list@redhat.com"
```

Comment:
Could we use `ceph-qe@redhat.com` instead of `ceph-qe-list@redhat.com`? This helps keep it within our QE team.